tests/datastructures/test_dsbsparse.py

```diff
@@ -173,7 +213,7 @@ def test_getitem(
         )
         dense = dsbsparse.to_dense()
 
-        reference = dense[..., *accessed_element]
+        reference = dense[(..., *accessed_element)]
```

Comment:
Why are all the indexed accesses below converted to tuples? It should be semantically the same, but was there some version incompatibility?